internal/envoy/v3/stats.go

```diff
@@ -148,8 +149,21 @@ func routeForAdminInterface(prefixes ...string) *envoy_filter_network_http_conne
 		config.RouteConfig.VirtualHosts[0].Routes = append(config.RouteConfig.VirtualHosts[0].Routes,
 			&envoy_config_route_v3.Route{
 				Match: &envoy_config_route_v3.RouteMatch{
-					PathSpecifier: &envoy_config_route_v3.RouteMatch_Prefix{
-						Prefix: prefix,
+					PathSpecifier: &envoy_config_route_v3.RouteMatch_Path{
+						Path: prefix,
```

Comment:
Let's rename `prefix` and `prefixes` to `path` and `paths` respectively and also update the godoc on this function to say "...requested paths..." instead of "...requested prefixes..." to avoid any future confusion.

Reply:
yeah good point fixed it. Renamed it to `p`  inside the for loop tho